src/components/WwwFrame/TheHeader.vue

```diff
@@ -314,10 +311,6 @@
 								class="header__button header__log-in"
 								:event="showPopupLogin ? '' : 'click'"
 								@click.native="auth0Login"
-								v-kv-track-event="[
-									['TopNav','click-Sign-in'],
```

Comment:
@edwinf-barreram we need to keep this one too please

Reply:
Ok, it is in another side, let me restore it!

Reply:
done!